### PR TITLE
Create user account in db:seed if it doesn't exist

### PR DESCRIPTION
**Pull request by ershad**
@smc/grandham protected `master` branch. Going forward all code changes will go through PRs. Also, each PR requires a green build and approval from another person to merge to master. Hope this would help to improve both discussions and code quality :) 

@tachyons please review?

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
-language = Language.where(name: 'Malayalam', short_code: 'ml', default: true).first_or_create!
+language = Language.where(name: "Malayalam", short_code: "ml", default: true).first_or_create!
```
Comment (redtachyons) [inline db/seeds.rb]: Can we set this rule in robocop if we prefer double quote over single quote?

Comment (maheshm) [reply db/seeds.rb]: Shouldnt it be the other way? Only string that require evaluation should be in double quote, right?

Comment (tachyons) [reply db/seeds.rb]: Yes, that is the default in rubocop. But that is opiniated, for instance rails prefers double quote by default.

Comment (ershad) [reply db/seeds.rb]: > Can we set this rule in robocop if we prefer double quote over single quote?
 
@redtachyons sure, let's do that in #39.

**Pull request merged**